src/main/java/com/labsynch/labseer/service/StandardizationServiceImpl.java

```diff
@@ -817,12 +807,10 @@ public String executeStandardization(String username, String reason) throws Stan
 		standardizationHistory.setStandardizationUser(username);
 		standardizationHistory.setStandardizationReason(reason);
 		standardizationHistory.merge();
-		this.reset();
```

Comment:
We reset these tables when doing dry run and I feel like its not necessary to reset them after doing standardization because they can be used to interrogate change later (at least it was helpful to me).